pkg/registry/apis/query/register.go

```diff
@@ -41,19 +41,19 @@ type QueryAPIBuilder struct {
 	returnMultiStatus      bool // from feature toggle
 	features               featuremgmt.FeatureToggles
 
-	tracer       tracing.Tracer
-	metrics      *metrics
-	parser       *queryParser
-	client       DataSourceClientSupplier
-	dryRunClient *DataSourceClientSupplier
-	registry     query.DataSourceApiServerRegistry
-	converter    *expr.ResultConverter
-	queryTypes   *query.QueryTypeDefinitionList
+	tracer            tracing.Tracer
+	metrics           *metrics
+	parser            *queryParser
+	client            DataSourceClientSupplier
+	passiveModeClient *DataSourceClientSupplier
```

Comment:
🤔  interesting -- this is comparing the individual datasource requests.  That will let us test the difference between multi-tenant and the direct plugin client request.

We also need a similar check for the difference between the whole lifecycle.  Essentially something to compare:  https://github.com/grafana/grafana/blob/main/pkg/api/ds_query.go#L41